guides/common/modules/proc_using-IdM-credentials-to-log-in-to-the-WebUI-with-a-Chrome-browser.adoc

```diff
@@ -0,0 +1,44 @@
+[id="Using_IdM_credentials_to_log_in_to_the_WebUI-with-a-Chrome-browser_{context}"]
+= Using IdM credentials to log in to the {ProjectWebUI} with a Chrome browser
+
+This section describes how to use a Chrome browser to log in to your {ProjectWebUI} with your {FreeIPA} (IdM) login and password.
+
+.Prerequisites
+* You have enrolled your {ProjectServer} into IdM and configured the server to use IdM for authentication.
+For more information, see {InstallingServerDocURL}Using_FreeIPA_{project-context}[Using {FreeIPA}] in _{InstallingServerDocTitle}_.
+* The host on which you are using the Chrome browser to log in to the {ProjectWebUI} is an IdM client.
+* You have a valid IdM login and password.
+* {Team} recommends using the latest stable Chrome browser.
+* An IdM server is running and reachable by the host.
+
+.Procedure
+. Enable the Chrome browser to use Kerberos authentication:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+google-chrome –auth-server-whitelist="*._example.com_" --auth-negotiate-delegate-whitelist=”*._example.com_"
```

Comment:
```suggestion
$ google-chrome –auth-server-whitelist="*._example.com_" --auth-negotiate-delegate-whitelist=”*._example.com_"
```

All commands in code blocks are generally prefixed with `#` for root users and `$` for non-root users